packages/components/molecules/f-media-element/stories/MediaElement.stories.js

```diff
@@ -10,11 +10,21 @@ export default {
     decorators: [withA11y]
 };
 
-export const MediaElementComponent = () => ({
+export const MediaElementComponent = (args, { argTypes }) => ({
     components: { MediaElement },
-    props: {
-    },
-    template: `<media-element />`
+    props: Object.keys(argTypes),
+    template: '<media-element v-bind="$props" />'
 });
 
 MediaElementComponent.storyName = 'f-media-element';
+
+MediaElementComponent.args = {
+    title: 'Stampcards',
+    text: 'See the stamps you’ve collected and any discounts you’ve earned.',
+    stacked: false,
+    reverse: true,
+    contentAlign: 'left',
+    imageAlign: 'center',
+    imageUrl: 'https://via.placeholder.com/250',
+    textSize: 'xxl'
```

Comment:
Can we use the `ALIGN` and `TEXT_SIZE` values from config.js to provide a list of selectable options for the controls?

Reply:
Will do, will add select boxes for font size and alignment options

Reply:
have added the better controls